source/administration/config-settings.rst

```diff
@@ -5210,6 +5212,12 @@ When running Mattermost on a single machine, this setting should always be enabl
 
 When running Mattermost in High Availablity mode, this setting should always be enabled. In a High Availability cluster, exactly one of the servers will be designated as the Scheduler at a time to ensure that duplicate tasks aren't created. See `High Availability documentation <https://docs.mattermost.com/deployment/cluster.html#job-server>`__ for more details.
 
+.. warning::
```

Comment:
The warning label is not showing up. Perhaps it's an alignment issue

Reply:
Should be fixed now, thanks!